.github/release-trigger.yml

```diff
@@ -1,2 +1 @@
 enabled: true
-multiScmName: bigframes
```

Comment:
I think we need to keep this, though I know we need to rename the build to python-bigquery-dataframes internally too.